package.json

```diff
@@ -106,9 +106,9 @@
 		"prettify": "prettier --write --ignore-path .gitignore \"**/*.{css,html,js,json,md,yaml,yml}\"",
 		"prettify:updated": "pipe-git-updated ---base=main -ext=css --ext=html --ext=js --ext=json --ext=md --ext=yaml --ext=yml -- prettier --write",
 		"test": "tad"
-	},
```

Comment:
This change should not be here

Reply:
No worries, change in other line was good. Anyway I've pushed it on my own and it's released now.

Again, thanks for helping on that